Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -501,7 +502,11 @@ public override List<Holding> GetAccountHoldings()
                 Connect();
             }
 
-            var holdings = _accountData.AccountHoldings.Select(x => ObjectActivator.Clone(x.Value)).Where(x => x.Quantity != 0).ToList();
+            var utcNow = DateTime.UtcNow;
+            var holdings = _accountData.AccountHoldings
+                .Select(x => ObjectActivator.Clone(x.Value))
+                .Where(x => x.Quantity != 0 && !OptionSymbol.IsOptionContractExpired(x.Symbol, utcNow))
```

Comment:
Maybe we could add a log letting the user now we are filtering out these expired option holdings